### DISCUSSION TITLE
Unable to connect Error: P1001: Can't reach database server

### DISCUSSION THREAD

**Discussion by conterio** [prisma/prisma#20203] [Q&A]
### Bug description

I'm just starting starting my application with t3 create.  I'm in the beginning stages of my application. I'm connecting to the database but am agetting the following error.

I'm on windows 10
connecting to a free postgres database on ElephantSQL

![image](https://github.com/prisma/prisma/assets/10100453/b3f3a8c5-c9d1-4f87-b3f2-e57ca3e91a93)

I have the connection string exactly as documented in primsa.

![image](https://github.com/prisma/prisma/assets/10100453/aa5c9a11-f1f0-48d4-b87e-5984f06aad84)

The kicker is i'm able to connect to the database with the same host, username, password, port, etc in dbeaver & pgadmin on the same machine.  There isn't a firewall issue or any inbound/outbound rules.

Everything i've read on this issue is the connection string is not formatted correctly but mine is.  I'm guessing it's something super simple i'm over looking but.

Thanks!

### How to reproduce

npx prisma db push

### Expected behavior

is able to connect to the db

### Prisma information
```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
    provider = "prisma-client-js"
    previewFeatures = ["jsonProtocol"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Example {
    id        String   @id @default(cuid())
    createdAt DateTime @default(now())
    updatedAt DateTime @updatedAt
}

// Necessary for Next auth
model Account {
    id                String  @id @default(cuid())
    userId            String
    type              String
    provider          String
    providerAccountId String
    refresh_token     String? // @db.Text
    access_token      String? // @db.Text
    expires_at        Int?
    token_type        String?
    scope             String?
    id_token          String? // @db.Text
    session_state     String?
    user              User    @relation(fields: [userId], references: [id], onDelete: Cascade)

    @@unique([provider, providerAccountId])
}

model Session {
    id           String   @id @default(cuid())
    sessionToken String   @unique
    userId       String
    expires      DateTime
    user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
}

model User {
    id            String    @id @default(cuid())
    name          String?
    email         String?   @unique
    emailVerified DateTime?
    image         String?
    accounts      Account[]
    sessions      Session[]
}

model VerificationToken {
    identifier String
    token      String   @unique
    expires    DateTime

    @@unique([identifier, token])
}

```
### Environment & setup

- OS: Windows 10
- Database: Postgresql
- Node.js version: 16.15.1

### Prisma Version

4.14.0

**Comment by ludralph** [comment]
Hi @conterio 👋 

Can you please enable [debugging](https://www.prisma.io/docs/concepts/components/prisma-client/debugging) and share the logs from the terminal?

**Reply by nurul3101** [reply]
@conterio Did you had a chance to look at Raphael's response? Please let us know in case you still need any help. If not, please consider marking the discussion as answered/closed.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
